src/fluid.rs

```diff
@@ -0,0 +1,434 @@
+extern crate image;
+extern crate rand;
```

Comment:
Those are not needed

Reply:
Oh, they are not needed after Rust 2018. OK, I removed them